minimesos/src/main/java/com/containersol/minimesos/mesos/MesosClusterContainersFactory.java

```diff
@@ -82,6 +83,15 @@ public void loadRunningCluster(MesosCluster cluster) {
                         case "master":
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 12 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1151)